client/post-editor/editor-revisions/index.jsx

```diff
@@ -72,29 +77,35 @@ EditorRevisions.propTypes = {
 	selectedRevisionId: PropTypes.number,
 	siteId: PropTypes.number.isRequired,
 
+	// connected to dispatch
+	recordTracksEvent: PropTypes.func.isRequired,
+
 	// localize
 	translate: PropTypes.func.isRequired,
 };
 
 export default flow(
 	localize,
-	connect( state => {
```

Comment:
Only the `recordTracksEvent` got added. Everything else is just a indentation change by prettier.